src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -646,7 +646,10 @@ def _get_custom_or_builtin_policy(cmd, client, name, subscription=None, manageme
     except (CloudError, HttpOperationError) as ex:
         status_code = ex.status_code if isinstance(ex, CloudError) else ex.response.status_code
         if status_code == 404:
-            return policy_operations.get_built_in(name)
+            try:
+                return policy_operations.get_built_in(name)
+            except CloudError:
```

Comment:
>CloudError [](start = 19, length = 10)

Can CloudError caused by other reason like internalServerError, if so, you may need to raise IncorrectUsageError for a more specific error code

Reply:
Thanks for your suggestion, I have improved the processing logic of this part.